src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+  @Override
+  public void doWork() {
+    System.out.println("Bulldozer started its work");
+  }
+
+  @Override
+  public void stopWork() {
+    System.out.println("Bulldozer stopped working");
```

Comment:
Similarly, the stop message should be more informative and indicate that it is a `Bulldozer` that has stopped working.